networking/multiple_networks/configuring-additional-network.adoc

```diff
@@ -139,13 +139,112 @@ include::modules/nw-multus-tap-object.adoc[leveloffset=+2]
+
+You must map the additional network to the OVN bridge to use it as an OVN-Kubernetes additional network. Bridge mappings allow network traffic to reach the physical network. A bridge mapping associates a physical network name, also known as an interface label, to a bridge created with OVS or OVN.
+
+You can create an `NodeNetworkConfigurationPolicy` object, part of the `nmstate.io/v1` API group, to declaratively create the mapping. This API is provided by the NMState Operator. By using this API, you can apply the bridge mapping to nodes that match the `nodeSelector` that you specify, such as `node-role.kubernetes.io/worker: ''` to apply to all worker nodes in your cluster.
+
+When attaching an additional network, you can either use the existing `br-ex` bridge or create a new bridge. Which approach to use depends on your specific network infrastructure.
+
+- If your nodes include only a single network interface, you must use the existing bridge. This network interface is owned and managed by OVN-Kubernetes and you must not remove it from the `br-ex` bridge or alter the interface configuration, or your cluster network will stop working correctly.
+- If your nodes include several network interfaces, you attach a different network interface to a new bridge, and use that for your additional network. This approach provides for traffic isolation from your primary cluster network.
+
+In the following example, the `localnet1` network is mapped to the `ovs-ex`:
+
+.Example mapping for sharing a bridge
+[source,yaml]
+----
+apiVersion: nmstate.io/v1
+kind: NodeNetworkConfigurationPolicy
+metadata:
+  name: mapping <1>
+spec:
+  nodeSelector:
+    node-role.kubernetes.io/worker: '' <2>
+  desiredState:
+    ovn:
+      bridge-mappings:
+      - localnet: localnet1 <3>
+        bridge: br-ex <4>
+        state: present <5>
+----
+<1> The name for the configuration object.
+<2> A node selector that specifies the nodes to apply the node network configuration policy to.
+<3> The name for the additional network from which traffic is forwarded to the OVS bridge. This additional network must match the name of the `metadata.name` field of the `NetworkAttachmentDefinition` object that defines the OVN-Kubernetes additional network.
+<4> The name of the OVS bridge on the node. This value is required only if you specify `state: present`.
+<5> The state for the mapping. Must be either `present` to add the bridge or `absent` to remove the bridge. The default value is `present`.
+
+In the following example, the `localnet2` network interface is attached to the `ovs-br1` bridge: Through this attachment, the network interface is available to the OVN-Kubernetes network plugin as an additional network.
+
+.Example mapping for nodes with multiple interfaces
+[source,yaml]
+----
+apiVersion: nmstate.io/v1
+kind: NodeNetworkConfigurationPolicy
+metadata:
+  name: ovs-br1-multiple-networks <1>
+  nodeSelector:
+    node-role.kubernetes.io/worker: '' <2>
```

Comment:
shouldn't this be in the `spec` ? The other yaml (sharing br-ex) is correct.

Reply:
@maiqueb, yes, it should!